frontend/src/core/services/auth.service.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { HttpService } from './http.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  public routePrefix = '/api/user';
+
+  constructor(
+    private httpService: HttpService
+  ) { }
+
+  loginUser(email: string, password: string):Observable<HttpResponse<{ assessToken: string }>> {
```

Comment:
Maybe it would be better to create DTOs so you can pass one object instead of bunch of strings